pkg/cmd/install/hubaddon/exec.go

```diff
@@ -77,6 +81,28 @@ func (o *Options) runWithClient() error {
 		if err != nil {
 			return fmt.Errorf("Error deploying %s CRDs: %w", addon, err)
 		}
+
+		// create namespace if its missing
+		clientSet, err := o.ClusteradmFlags.KubectlFactory.KubernetesClientSet()
+		if err != nil {
+			return fmt.Errorf("failed to create kubernetes clientSet")
+		}
+
+		ns, err := clientSet.CoreV1().Namespaces().Get(context.Background(), o.values.Namespace, metav1.GetOptions{})
+		if err != nil && errors.IsNotFound(err) {
+			ns, err = clientSet.CoreV1().Namespaces().Create(context.Background(), &corev1.Namespace{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: o.values.Namespace,
+				},
+			}, metav1.CreateOptions{})
+			if err != nil {
+				return fmt.Errorf("failed to create namespace %s: %w", ns, err)
+			}
+
```

Comment:
Sorry to have a formatting nitpick, but can you remove this newline?

Reply:
removed 😄